src/countryCodes.ts

```diff
@@ -0,0 +1,257 @@
+import { CountryCode } from './generated/graphql'
```

Comment:
Not a big fan to put that at the root
Why did you remove it from public ?
Alternative could be in `core`

Reply:
Core sound like a good alternative,

updated!